src/Agent/NewRelic/Profiler/Profiler/CorProfilerCallbackImpl.h

```diff
@@ -1147,7 +1148,6 @@ namespace NewRelic { namespace Profiler {
             nrlog::StdLog.SetAzureFunctionMode(_systemCalls->IsAzureFunction());
             nrlog::StdLog.SetAzureFunctionLogLevelOverride(_systemCalls->IsAzureFunctionLogLevelOverrideEnabled());
             nrlog::StdLog.SetEnabled(_systemCalls->GetLoggingEnabled(configuration->GetLoggingEnabled()));
-            nrlog::StdLog.SetInitalized();
```

Comment:
It looks like removing this line from here is breaking console logging in the profiler, because the logger is never set to "initialized".

Reply:
Good catch. I moved it outside the `if` block.